### PR TITLE
fix!: Make Tabs correctly handle a single Tab Button

### PR DESCRIPTION
**Pull request by RubenSibon**
# Describe the pull request

A fix for issue #1774 that Tabs List gives an breaking error when there is only one, non-iterable element in the Tabs List.

## What

A Tabs List does not always expect multiple React Node's.

## Why

Tabs (List) should be able to handle a single Tab button.

## How

Remove the map method and just returning the value of `children`.

## Checklist

Before submitting your pull request, please ensure you have done the following. Check each checkmark if you have done so or if it wasn't necessary:

- [x] Add or update stories (tested with stories)
- [x] Manually tested in Prototypes (Amopis)
- [x] Start the PR title with a Conventional Commit prefix, [as explained here](https://github.com/Amsterdam/design-system/blob/main/documentation/publishing.md?plain=1#L11).

## Additional notes

- Issue: #1774

### CODE REVIEW

File: packages/react/src/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ const TabsRoot = forwardRef(
 
     const allTabs = useMemo(() => {
       if (!Array.isArray(children)) return []
-      return (children[0].props.children as ReactNode[]).map((child) => child)
+      return children[0].props.children
```
Comment (VincentSmedinga) [inline packages/react/src/Tabs/Tabs.tsx:28]: Do we remember why this casting was done in the first place? (I don’t.)

Comment (RubenSibon) [reply packages/react/src/Tabs/Tabs.tsx:28]: Nope, I don't know either. Removing it doesn't seem to be break anything.

**Comment by RubenSibon** [comment]
Tested it again in the Tabs Story and in the existing Tabs in Amopis in Prototypes.

File: packages/react/src/Tabs/Tabs.tsx (hunk 1 above)
Comment (VincentSmedinga) [reply packages/react/src/Tabs/Tabs.tsx:28]: We might be bumping into React internals here. This change does indeed fix it. We discussed that we should not let `Tabs` interfere with children of `Tabs.List`; I’ll add a ticket for it.

**Review by VincentSmedinga — APPROVED** [review]

**Pull request merged**